lib/utf8.pm

```diff
@@ -1,17 +1,14 @@
-package utf8;
+package utf8 1.29;
 
 # This file only defines the import/unimport subs, the rest are implemented by
 # always-present functions in the perl interpreter itself.
 # See also `universal.c` in the perl source
 
-use strict;
-use warnings;
+use v5.38;
```

Comment:
Earlier for `overload.pm` you picked 5.40 - any reason not to do that here? As far as I can see, the only difference is it adds `feature 'try'` which shouldn't be a problem here.

I think we should consistently pick a horizon for all these modernisations - 5.40 seems good, as it's just before 5.42 performed a few removals (such as apostrophe-in-package and switch), which might cause us some nontrivial issues.

Reply:
Sorry, yes, I should explain my thinking. With overloading I picked 5.40 as that was the lowest that exported builtin which the test utilised and I wanted to make consistent with the code under test. For utf8 however I picked 5.38 as it was the lowest that gave the module_true feature.

In general I've mostly tried to pick the lowest I can get away with for the features I wanted but I'm happy to switch that approach.

Reply:
I think as an overall process it'd be good to `use v5.40` in most files and actually use the features from them where appropriate. It'll take us a while to get around to doing that everywhere, so 5.40 seems a good point to pick. By the time that's all done it'll be time to go again for 5.50 ;)